Gruntfile.js

```diff
@@ -135,6 +135,18 @@ module.exports = function(grunt) {
             }
         },
 
+        postcss: {
+            options: {
+                map: false,
+                processors: [
+                    require('autoprefixer')
```

Comment:
@ZachTRice I would add the require to the top of the `Gruntfile` 

[some reasoning](http://justbuildsomething.com/node-js-best-practices/#2)